browser/src/libs/options/OptionsMenu.tsx

```diff
@@ -74,19 +74,41 @@ export const OptionsMenu: React.FunctionComponent<OptionsMenuProps> = ({
             !currentTabStatus.hasPermissions &&
             !PERMISSIONS_PROTOCOL_BLACKLIST.includes(currentTabStatus.protocol) && (
                 <div className="options-menu__section">
-                    <div className="alert alert-danger">
-                        Sourcegraph is not enabled on <strong>{currentTabStatus.host}</strong>.{' '}
-                        <a
-                            href=""
+                    <div className="alert alert-info">
+                        <p>
+                            The Sourcegraph browser extension adds hover tooltips to code views on code hosts such as
+                            GitHub, GitLab, Bitbucket Server and Phabricator.
+                        </p>
+                        <p>
+                            You must grant permissions to enable Sourcegraph on <strong>{currentTabStatus.host}</strong>
+                            .
+                        </p>
+                        <button
+                            type="button"
+                            className="btn btn-outline-info request-permissions__test"
```

Comment:
I personally liked the `btn-secondary` in Rob's screenshot more, what do others think?
```suggestion
                            className="btn btn-secondary request-permissions__test"
```

Reply:
`btn-light` is actually what we'd want here:

![image](https://user-images.githubusercontent.com/1741180/81801060-21f7e480-9514-11ea-9c4e-218dc3fe4d2e.png)

Reply:
```suggestion
                            className="btn btn-light request-permissions__test"
```